### PR TITLE
Moving the update_in_comment option from the process param into the config file.

### PR DESCRIPTION
**Pull request by cropalato**
I really like that feature, thanks for your PR, But I agree with @MytkoEnko ( https://github.com/prometheus-community/jiralert/pull/161#issuecomment-1580800245 ). Should be more flexible if we use config file to manage that.
So, here is my 2 cents...

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -149,6 +149,9 @@ type ReceiverConfig struct {
 	// Label copy settings
 	AddGroupLabels bool `yaml:"add_group_labels" json:"add_group_labels"`
 
+	// Flag to auto-resolve opened issue when the alert is resolved.
```
Comment (twotired) [inline pkg/config/config.go]: this should be "// Flag to enable updates in comments"

File: pkg/config/config_test.go [hunk 2]
```diff
@@ -385,6 +388,8 @@ func newReceiverTestConfig(mandatory []string, optional []string) *receiverTestC
 		var value reflect.Value
 		if name == "AddGroupLabels" {
 			value = reflect.ValueOf(true)
+    } else if name == "UpdateInComment" {
```
Comment (twotired) [inline pkg/config/config_test.go]: indentation appears inconsistent

**Review by twotired — CHANGES_REQUESTED** [review]
there are just a couple minor issues but otherwise LGTM

**Comment by cropalato** [comment]
Fixing small issues:

- Adding '//' flag to enable updates in comments
- Fixing indentation

**Comment by cropalato** [comment]
Fixed.

**Comment by cropalato** [comment]
@twotired Do you want me to change something else?

**Pull request merged**